### DISCUSSION TITLE
How to add type extensions in fusion gateway?

### DISCUSSION THREAD

**Discussion by liyanjie2048** [ChilliCream/hotchocolate#6985] [Q&A]
I'm new to learning Hotchocolate Fusion. I want to know how stitch multiple schemas from different service and extend types in this situation:

```
type User // from A service
{
    Id
    ...
}
type Order // from B service
{
    UserId
    ...
}
```

I wonder how to add "Orders" to "User" type and "User" to "Order" type and query data.
Is there any way like "AddTypeExtensions" in Stitching Gateway or docs to learn?

**Comment by PascalSenn** [comment]
@liyanjie2048  
In fusion your Order service would have a field called `user` of type `User`. 
service A
```graphql
type User
{
    id: ID!
    name: String
}
```
serivce b
```graphql

type Order
{
    id: ID!
    user: User
}
type User
{
    id: ID!
    name: String
}
```
